JavaScript/call-apply-bind.md

```diff
@@ -0,0 +1,148 @@
+# Call(), Apply() & Bind()
+___
+
+#### In essence, the purpose of the Call(), Apply(), Bind() methods is to enable the ability to control what the ```this``` variable ends up being when a function is run.
```

Comment:
Reads better 'Essentially the purpose of call(), apply() and bind() are to enable 'this', to be better controlled when a function isexecuted'